app/controllers/emails_controller.rb

```diff
@@ -7,7 +7,7 @@ def show
     @templates = EmailTemplate.with_status(get_status)
     @addresses = @event.email_addresses_of_type(get_status)
 
-    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses,
+    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses.join(','),
```

Comment:
Hm, this is more refactoring than I'd like in a PR for another topic. I'd like some feedback from somebody else, maybe from your team or from the email people @frederike-ramin @aloeser @Baschdl. I would really like this to be separated from this PR. It shouldn't be too much effort with `git stash`.